package.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "@demos-europe/demosplan-ui",
-  "version": "0.0.2",
+  "name": "demosplan-ui",
+  "version": "1.0.0",
   "license": "MIT",
   "private": false,
-  "description": "Vue components, Vue directives, Design Token and Scss files to build interfaces for demosPlan.",
+  "description": "demosplan-ui is a vue component library",
```

Comment:
why not rolling back?

Reply:
This happens here, sorry https://github.com/demos-europe/demosplan-ui/pull/13/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519